README.md

```diff
@@ -2,6 +2,8 @@
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE) ![Version](https://img.shields.io/nuget/v/MessageDialogService?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/MessageDialogService?style=flat-square)
 
+⚠ This readme is applicable for versions 1.0.X and below.
```

Comment:
This readme is fine for version 2. The one that is deprecated (only applies on version 1) is the `doc/GettingStartedUWP.md`.

Reply:
So you can just remove that change.

Reply:
I'll remove the references for uwp from readme then